hbp_knowledge/proteostasis/brehme2016.bel

```diff
@@ -226,32 +267,42 @@ p(HGNC:HSPB1) -- a(HBP:"amyloid-beta aggregates")
 p(HGNC:CRYAB) -- a(HBP:"amyloid-beta aggregates")
 p(HGNC:HSPB6) -- a(HBP:"amyloid-beta aggregates")
 p(HGNC:HSPB8) -- a(HBP:"amyloid-beta aggregates")
+UNSET Confidence
 
 SET Evidence = "Furthermore, sHSPs
 were found to be consistently upregulated in the aging human brain
 and in the context of neurodegenerative diseases (Brehme et al.,
 2014). "
+SET Confidence = "High"
 SET MeSHAnatomy = "Brain"
 bp(GO:aging) -> p(FPLX:HSPB)
 path(MESH:"Alzheimer Disease") -> p(FPLX:HSPB)
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Therapeutic approaches to
 overcome proteostasis deficiencies have largely focused on the
 activation of HSF1, the heat shock transcription factor responsible
 for simultaneous upregulation of the expression of multiple
 molecular chaperones during stress (Calamini et al., 2011; Pierce
 et al., 2013)."
+SET Confidence = "High"
 p(HGNC:HSF1) -> bp(HBP:Proteostasis)
+UNSET Confidence
 
 SET Evidence = "Multiple studies in model systems demonstrate
 that overexpression of HSP70 can reduce toxicity and protein
 aggregation."
+SET Confidence = "High"
 p(FPLX:HSPA) -| path(HBP:proteotoxicity)
 p(FPLX:HSPA) -| a(MESH:"Protein Aggregates")
+UNSET Confidence
 
+#which chemical used? No mention in evidence sentence
 SET Evidence = "Compounds such as the allosteric inhibitor of ATP-binding for the
 inducible HSP70 isoform HSPA1A/HSPA1B, called HS-72, will
 likely lead to beneficial consequences (Howe et al., 2014)."
+SET Confidence = "Low"
```

Comment:
allosteric inhibitor of ATP-binding for the inducible HSP70 isoform HSPA1A/HSPA1B, called **HS-72**
HS-72 has PUBCHEM ID 40009526